bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/ReolinkHandler.java

```diff
@@ -115,6 +115,10 @@ public void channelRead(@Nullable ChannelHandlerContext ctx, @Nullable Object ms
                     break;
                 case "/api.cgi?cmd=GetAiState":
                     ipCameraHandler.setChannelState(CHANNEL_LAST_EVENT_DATA, new StringType(content));
+                    if (content.contains("\"detail\": \"not support\"")) {
```

Comment:
I believe this is not the best way forward to remove the NPE.
1. It has to parse the string even when there is no issue and this takes time to process.
2. Your NPE is triggered by "not support" and peoiple on the forum are reporting they get the same NPE because the camera reports "device offline". Can you test a fix that checks for NULL as that will be faster code to run and will work for both use cases and any others which are not yet known?

Reply:
Consider using the following in case the camera does not reply with a valid json reply... 
```
try {
code here....
 } catch (JsonParseException e) {
}
```

also checks for ```aiResponse[0].value == null```

Reply:
A fix for just the NPE has been merged so this will mean a merge conflict unless you remove this from your branch (probably the easiest way) or do a rebase and fix the conflict. Looking forward to having your changes for channel 0 merged when you have time to finish this off.

Reply:
I took care of the merge conflict to make it easier, so will resolve this comment and then it just leaves the DCO failure to fix and some minor readme changes to look at so we can merge.